src/main/java/io/wisetime/connector/IntegrateApplication.java

```diff
@@ -57,10 +58,23 @@ private void addEndpoints() {
 
     post("/receiveTimePostedEvent", (request, response) -> {
       TimeGroup userPostedTime = om.readValue(request.body(), TimeGroup.class);
-      // TODO: Handle PostResult from connector
-      wiseTimeConnector.postTime(request, userPostedTime);
+
+      PostResult postResult = wiseTimeConnector.postTime(request, userPostedTime);
+
       response.type("plain/text");
-      return "success";
+      switch (postResult) {
+        case SUCCESS:
+          response.status(200);
+          return "success";
```

Comment:
Can you capitalise "Success" for consistency with other return values? Sorry, my OCD ;)